apps-rendering/package.json

```diff
@@ -34,6 +34,8 @@
 	"license": "ISC",
 	"prettier": "@guardian/prettier",
 	"dependencies": {
+		"@aws-sdk/credential-providers": "3.341.0",
+		"@aws-sdk/client-ssm": "3.441.0",
```

Comment:
Are these meant to be a 100 minor versions apart?

Reply:
My bad. It was a typo while adding versions manually.

Fixed in https://github.com/guardian/dotcom-rendering/pull/9411/commits/32c9a6cf38ce179333e14264fcd54c2106e24878